src/Resources/values-tw/strings.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string name="app_name">就近共享</string>
+	<string name="app_titlebar_title_default">傳輸資料到您的 Windows 10 / 11 電腦</string>
+	<string name="app_titlebar_title_receive">從您的 Windows 10 / 11 電腦接收資料</string>
+	<string name="app_titlebar_title_receive_setup">設定與資訊</string>
+
+	<string name="assets_prefix">zh</string>
```

Comment:
Please follow the [Wiki](https://github.com/nearby-sharing/android/wiki/Contributing#add-translation) to add the missing translations and set this `assets_prefix` to the correct language-prefix afterwards